### PR TITLE
Add members to organization, test organization membership view and or…

### LINKED ISSUE #124: Organization has many members
For #26.
- [x] Test organization view
- [x] Test organization membership view
- [x] Add actual code

### PR DESCRIPTION
**Pull request by joshsmith**
Closes #124.

### CODE REVIEW

**Comment by snewcomer** [comment]
@JoshSmith is there any migration file work that needs to happen? indexes perhaps?

**Comment by joshsmith** [comment]
@snewcomer this doesn't touch models, just views. The models are already done.

**Comment by npendery** [comment]
This should be closing #124 not #123, correct? Related to #123 though

**Comment by joshsmith** [comment]
That's what it says, doesn't it?

**Comment by npendery** [comment]
Now it does 😏

**Comment by joshsmith** [comment]
Caught me! 🔴 ✋

File: test/views/organization_membership_view_test.exs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+  # Bring render/3 and render_to_string/3 for testing custom views
+  import Phoenix.View
+
+  test "renders all attributes and relationships properly" do
+    organization = insert(:organization)
+    user = insert(:user)
+    organization_membership = insert(:organization_membership, member: user, organization: organization)
+
+    organization_membership =
```
Comment (npendery) [inline test/views/organization_membership_view_test.exs:16]: What is the reason for the reassignment? Do the relationships not get loaded through the factory output?

**Comment by npendery** [comment]
Looks good! 👍

File: test/views/organization_membership_view_test.exs (hunk 1 above)
Comment (joshsmith) [reply test/views/organization_membership_view_test.exs:16]: I think that's right, yes.

**Pull request merged**